ci/scripts/java_build.sh

```diff
@@ -23,6 +23,61 @@ source_dir=${1}/java
 cpp_build_dir=${2}/cpp/${ARROW_BUILD_TYPE:-debug}
 with_docs=${3:-false}
 
+if [[ "$(uname -s)" == "Linux" ]] && [[ "$(uname -m)" == "s390x" ]]; then
+  # Since some files for s390_64 are not available at maven central,
+  # download pre-build files from bintray and install them explicitly
+  mvn_install="mvn install:install-file"
+  wget="wget"
+  bintray_base_url="https://dl.bintray.com/apache/arrow"
+
+  bintray_dir="flatc-binary"
+  grp="com.github.icexelloss"
+  artifact="flatc-linux-s390_64"
+  ver="1.9.0"
+  extension="exe"
+  target=${artifact}-${ver}.${extension}
+  ${wget} ${bintray_base_url}/${bintray_dir}/${ver}/${target}
+  ${mvn_install} -DgroupId=${grp} -DartifactId=${artifact} -Dversion=${ver} -Dpackaging=${extension} -Dfile=`pwd`/${target}
```

Comment:
Generally, `$(...)` is preferred than `` `...` ``:

```suggestion
  ${mvn_install} -DgroupId=${grp} -DartifactId=${artifact} -Dversion=${ver} -Dpackaging=${extension} -Dfile=$(pwd)/${target}
```